### PR TITLE
chore: cleanup this repo

### PR DESCRIPTION
**Pull request by fmvilas**
**Description**

- Removes all version files and leaves only the current spec version.
- Removes all schema.json files and place links on the README.
- Creates a README file on the examples folder.
- Removes tests, except `mdv`.
- Removes Travis.

### CODE REVIEW

**Comment by derberg** [comment]
what do you mean that `mdv` tests do not work, they worked fine with https://github.com/asyncapi/spec/blob/master/.github/workflows/if-nodejs-pr-testing.yml worklow and this fix https://github.com/asyncapi/spec/commit/6cf2100ca81add412cf9ad9b69e19d9c14da0f40

File: README.md [hunk 1]
```diff
@@ -54,12 +54,14 @@
...
-* [Version 2.0.0](/versions/2.0.0/asyncapi.md) — [Machine-readable version](/versions/2.0.0/schema.json) (latest stable) 
-* [Version 1.2.0](/versions/1.2.0/asyncapi.md) — [Machine-readable version](/versions/1.2.0/schema.json)
-* [Version 1.1.0](/versions/1.1.0/asyncapi.md) — [Machine-readable version](/versions/1.1.0/schema.json)
-* [Version 1.0.0](/versions/1.0.0/asyncapi.md) — [Machine-readable version](/versions/1.0.0/schema.json)
+* [Version 2.0.0](https://github.com/asyncapi/spec/blob/2.0.0/versions/2.0.0/asyncapi.md) (latest)
+* [Version 1.2.0](https://github.com/asyncapi/spec/blob/1.2.0/README.md) (deprecated)
+* [Version 1.1.0](https://github.com/asyncapi/spec/blob/1.1.0/README.md) (deprecated)
+* [Version 1.0.0](https://github.com/asyncapi/spec/blob/1.0.0/README.md) (deprecated)
+
+**Looking for machine-readable files?** Check out our [asyncapi-node](https://www.github.com/asyncapi/asyncapi-node) repo.
```
Comment (derberg) [inline README.md:64]: I would just be explicit by writing `json schema`

**Comment by fmvilas** [comment]
@derberg `mdv` is always passing because it's not testing anything 😄 That's what I mean by "it's not working".

**Comment by fmvilas** [comment]
Left you a comment here: https://github.com/Mermade/mdv/pull/49

**Comment by derberg** [comment]
@fmvilas oh man, you need to try to be more descriptive 😆 "it's not testing anything"? how come if on your other PR I see https://github.com/asyncapi/spec/pull/511/checks?check_run_id=2112204488 ? what do you mean not testing? like it picks up a file and says it is ok, but it isn't? or what?

**Comment by fmvilas** [comment]
> like it picks up a file and says it is ok, but it isn't? or what?
Exactly that! 😄 

It's never getting into this function: https://github.com/Mermade/mdv/blob/master/mdv.js#L40.

**Comment by fmvilas** [comment]
Look: https://www.loom.com/share/259371eab9a14a0c8cbaf7f6a35af432

**Comment by derberg** [comment]
@fmvilas oh 😞 I looked into the source code of glob but damn, I ran away from there as I was afraid for my mental health. I'll try on a better day than Monday

**Comment by fmvilas** [comment]
It's solved already! :) https://github.com/Mermade/mdv/pull/49#issuecomment-799576145

File: README.md [hunk 2]
```diff
@@ -54,12 +54,14 @@
   :raised_hands: <a href="https://opencollective.com/asyncapi">Become a sponsor</a> :raised_hands:
 </p>
 
-## The specification
+## Read the specification
```
Comment (derberg) [inline README.md:57]: specification working document should also be listed here in the readme, I mean the `asyncapi.md` that is on root on master (after merge) to make it clear where is the current not stable version

Comment (fmvilas) [reply README.md:57]: Done!

**Review by derberg — APPROVED** [review]
I think it was fine having it on the root of the folder, no need to move it to `spec` dir in spec repo 
Screenshot 2021-03-15 at 18 46 19

approving anyway

**Comment by fmvilas** [comment]
Got the idea from GraphQL repo: https://github.com/graphql/graphql-spec/tree/main/spec. Our spec is already big and would be great to split it into several documents to ease the edition. I know the URL is not great but not that bad. Thanks for approving man!

**Comment by derberg** [comment]
It is actually not included in the release. This was an unintentional release caused by the release automation initial run. 1.0 version of the spec was released a long time ago. The release mentioned by the bot was removed.

**Comment by derberg** [comment]
Not included in release 2.1 yet, 2.1 was mistakenly triggered by our automation.

**Comment by derberg** [comment]
Not included in release 2.1 yet, 2.1 was mistakenly triggered by our automation.

**Pull request merged**